src/HasStates.php

```diff
@@ -140,6 +140,21 @@ public function transitionTo($state, string $field = null)
         $this->{$field}->transitionTo($state);
     }
 
+    /**
+    * @param \Spatie\ModelStates\State|string $state
+    * @param string $fromClass
+    */
+    public function getAllowedTransitionsFrom($state, string $fromClass)
```

Comment:
Not sure about the name (see comments further down), also use a return type hint here and remove the docblocks